tests/test_main.py

```diff
@@ -15,3 +15,14 @@ def test_when_calling_main_without_arguments_then_usage(self):
         with self.assertRaisesRegex(Exception,
                                     'USAGE: %s results.tap' % __file__):
             main([__file__])
+
+    @unittest.expectedFailure
+    def test_produce_traceback_output(self):
+        """Add traceback on TAP ouput"""
+        main([__file__])
```

Comment:
Here maybe just update the comment. This test is not useful by itself for unit testing purposes but is useful b/c later we'll call the program to submit the results to Kiwi TCMS and if there is an error in the traceback handling code the program may blow up.

This needs to be easily conceivable from the comment or from the test method name otherwise the next time we read this we are not going to know.